src/ImageGallery.jsx

```diff
@@ -634,8 +636,10 @@ export default class ImageGallery extends React.Component {
       }
 
       let totalThumbnails = this._thumbnails.children.length;
+      let lastThumbnailOffset = this.props.lastThumbnailOffset > 0 ? this.props.lastThumbnailOffset : 1;
+
       // scroll-x required per index change
-      let perIndexScroll = totalScroll / (totalThumbnails - 1);
+      let perIndexScroll = totalScroll / (totalThumbnails - lastThumbnailOffset);
```

Comment:
What happens if totalThumbnails is 2 and lastThumbnailOffset is 3?

Reply:
How about 

```javascript
 let perIndexScroll = Math.max(0, totalScroll / (totalThumbnails - lastThumbnailOffset));
```